app/controllers/submit.js

```diff
@@ -476,43 +499,46 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
         const reviewsState = this.get('model.reviewsState');
         const moderationType = this.get('moderationType');
         if (reviewsState && moderationType) {
-            return EDIT_MESSAGES.line2[reviewsState][moderationType];
+            let message = EDIT_MESSAGES.line2[reviewsState][moderationType];
+            if (reviewsState === 'rejected' && moderationType === 'pre-moderation' && !(this.get('isAdmin'))) {
```

Comment:
If this is a preprint that needs to be resubmitted, but you are not an admin, you should see a message that the admin contributor needs to do this.